pkg/virt-operator/resource/apply/instancetype_test.go

```diff
@@ -62,26 +72,22 @@ var _ = Describe("Apply Instancetypes", func() {
 		})
 
 		It("should create instancetype", func() {
-			getCalled := expectGetReturnNotFound(fakeClient, apiinstancetype.ClusterPluralResourceName)
 			createCalled := expectCreate(fakeClient, apiinstancetype.ClusterPluralResourceName, instancetype)
 			Expect(reconciler.createOrUpdateInstancetype(instancetype)).To(Succeed())
-			Expect(*getCalled).To(BeTrue())
```

Comment:
As a follow up, I think you can reduce the complexity of `expectCreate` (or rather fix?) by interrogating the fake client if it contains them...